src/browser/app/Header.react.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component, PropTypes } from 'react';
+import Link from './components/Link.react';
```

Comment:
In React Native, we have native/app/components. I think it belongs there, but I can change my mind. The reason why I prefer app/components, it's because it the implementation detail.